lib/parser.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Parser do
+  def html_links(document) do
+    Floki.parse_document!(document)
+    |> Floki.find("a")
+    |> Floki.attribute("href")
```

Comment:
This is actually interestingThe way how Floki operates on collections is interesting. I like it.